ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -390,6 +393,25 @@ private function formatLinksColumn($column, $data): array {
     return $out;
   }
 
+  /**
```

Comment:
To meet the full standard we should have a one-liner description for the function - in this case of this function it probably would make it clearer too

Non blocking

Reply:
Updated